scripts/Muon/GUI/Common/contexts/muon_context.py

```diff
@@ -32,12 +34,25 @@ def __init__(self, muon_data_context=None, muon_gui_context=None,
         self.fitting_context = fitting_context
         self.base_directory = base_directory
         self.workspace_suffix = workspace_suffix
-        self.ads_observer = MuonContextADSObserver(self.remove_workspace_by_name, self.clear_context)
+        self._frequency_context = frequency_context
 
-        self.gui_context.update({'DeadTimeSource': 'None', 'LastGoodDataFromFile': True, 'selected_group_pair': ''})
+        self.ads_observer = MuonContextADSObserver(
+            self.remove_workspace_by_name,
+            self.clear_context)
+
+        self.gui_context.update(
+            {'DeadTimeSource': 'None',
```

Comment:
Can these all be in line with each other?

Reply:
this was autopep 8 as flake8 does not like lines to get too long.

Reply:
Sorry I wasn't being clear, I mean can the all the elements of the dictionary have the same indentation. Lines 45 and 46 are indented to line up with the bracket at the end of line 43, but 44 is only indented one tab out from line 43.

Reply:
flake8 will catch those sort of things